plugins/algebra/include/detray/array/include/plugins/algebra/array_definitions.hpp

```diff
@@ -8,11 +9,11 @@ namespace detray {
     using scalar = algebra::scalar;
 
     template <typename value_type, unsigned int kDIM>
-    using darray = algebra::array_s<value_type, kDIM>;
+    using darray = std::array<value_type, kDIM>;
```

Comment:
Exactly! This keeps darray fixed to std::array, even if we set the global containers to vecmem containers.